shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/manual/postgresql/uuid_primary_key/sql.xml

```diff
@@ -0,0 +1,52 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<command>
+    <create-table-order>
+        CREATE TABLE test.t_order (
+        id varchar(255) NOT NULL,
+        order_id int4 NOT NULL,
+        user_id int NOT NULL,
+        PRIMARY KEY (id)
+        )
+    </create-table-order>
+    
+    <create-table-order-item>
+        CREATE TABLE test.t_order_item (
+        item_id int8 NOT NULL,
+        order_id int4 NOT NULL,
+        user_id int8 NOT NULL,
+        status varchar(50),
```

Comment:
Could we share the same `sql.xml` between databases for `uuid_primary_key`?

And it's better to use `text_primary_key` etc.

Reply:
I will extract the same statement at next pr

Reply:
OK. Could we just simplify the create table SQL in this case?

Reply:
Yes, i will remove useless statement.